.github/drafts/15938/15938_Audit_Logs_Outline.md

```diff
@@ -0,0 +1,108 @@
+# Audit logs
+
+Audit logs are a vital way to keep records of user activity related to app changes, platform settings, and sensitive data. With access to this information, you can ensure that you're able to easily follow up and address any unexpected or undesirable events.
+
+{Screenshot of audit logs page}
+
+---
+
+## Logged events
+
+(Should we include the technical names of the events? Or maybe one example event per type? i.e. `datasource.created` etc.)
```

Comment:
Examples of events are not necessary here in the beginning as it will be too much info for the user.  We should have an exhaustive list probably before Searching audit logs

Reply:
In the Introduction section of the doc, we only want to give them an overview of what kind of information will be accessible in the audit logs.  I think post this is where we should show users how to access Audit Logs on the Appsmith platform.